website/content/en/docs/Configure/custom_cloud.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Pod Identity in Custom Cloud"
+linkTitle: "Pod Identity in Custom Cloud"
+weight: 5
+description: >
+  Using AAD Pod Identity in custom azure cloud environment.
+---
+
+This document highlights the steps to configure and use AAD Pod Identity in custom azure cloud environments.
+
+1. Ensure the cloud name in `/etc/kubernetes/azure.json` is set to `AzureStackCloud`
+
+    ```json
+    {
+        "cloud": "AzureStackCloud",
+        "tenantId": "xxxx",
+        "subscriptionId": "xxxx",
+        ...
+    }
+    ```
+
+2. Mount the JSON file that contains the custom cloud environment details. The custom cloud environment file is stored in the file system of the Kubernetes node. The `go-autorest` library is configured to [read the azure environment from file](https://github.com/Azure/go-autorest/blob/master/autorest/azure/environments.go#L239-L241) by default for `AzureStackCloud`
```

Comment:
Using tag instead of master in the URL in case the file gets modified in the future.

```suggestion
2. Mount the JSON file that contains the custom cloud environment details. The custom cloud environment file is stored in the file system of the Kubernetes node. The `go-autorest` library is configured to [read the azure environment from file](https://github.com/Azure/go-autorest/blob/autorest/v0.10.0/autorest/azure/environments.go#L219-L221) by default for `AzureStackCloud`
```